### DISCUSSION TITLE
Simple filesystem-browser

### DISCUSSION THREAD

**Discussion by Heathcliff77** [alphapapa/listen.el#23] [Ideas]
Hello

I would like to have a simple filesystem-browser (like dired for example) for navigating around my music and adding it to queues.
You could say that this is opposed to the "library" function/concept, which constructs a view of music (the filesystem) based on genres, artists, albums et.c.

This video is a good demonstration of what I am imagining: https://youtu.be/m8pvj4QQU9w?t=374 (start at 6:14)

I think some development might be required to get a simple filesystem-browser?

Best Regards

**Comment by alphapapa** [comment]
Hello,

What's wrong with using Dired?

**Reply by Heathcliff77** [reply]
Well, how would that work?

From a dired buffer,  how would I add something (single file, directory, playlist-file (.m3u, .pls et.c.) ) to a listen-queue?

**Reply by alphapapa** [reply]
Please see the documentation: https://github.com/alphapapa/listen.el#adding-tracks-to-a-queue
